app/views/insured/employee_roles/edit.html.haml

```diff
@@ -5,7 +5,7 @@
     .row
       = form_for(@person, employee_role_submission_options_for(@person)) do |f|
         .col-lg-7.col-md-7.col-sm-8.col-xs-12.bottom-pd
-          %h3 Contact Information
+          %h3= l10n("contact_information")
```

Comment:
This should be l10n(".contact_information"), which will infer the path "insured.employee_roles.edit.contact_information". In other words, in this file, you would put l10n(".contact_information"), and in your English translation file, you would put "en.insured.employee_roles.edit.contact_information"